dev/rpm/build-rpm.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Build an RPM package for Volta
+
+# using the directions from https://rpm-packaging-guide.github.io/
+
+# exit on error
+set -e
+
+# only argument is the version number
+release_version="${1:?Must specify the release version, like \`build-rpm 1.2.3\`}"
+archive_filename="v${release_version}.tar.gz"
+
+# make sure these packages are installed
+# (https://rpm-packaging-guide.github.io/#prerequisites)
+sudo yum install gcc rpm-build rpm-devel rpmlint make python bash coreutils diffutils patch rpmdevtools
+
+# set up the directory layout for the RPM packaging workspace
+# (https://rpm-packaging-guide.github.io/#rpm-packaging-workspace)
+rpmdev-setuptree
+
+# create a tarball of the repo for the specified version
+# using prefix because the rpmbuild process expects a 'volta-<version>' directory
+# (https://rpm-packaging-guide.github.io/#putting-source-code-into-tarball)
+git archive --format=tar.gz --output=$archive_filename --prefix="volta-${release_version}/" "v${release_version}"
+
+# move the archive to the SOURCES dir
+# (https://rpm-packaging-guide.github.io/#working-with-spec-files)
+mv "$archive_filename" "$HOME/rpmbuild/SOURCES/"
```

Comment:
Does `rmpdev-setuptree` clean out anything in these directories as well? If not, we might want to do that just to make sure we don't get contamination from old builds.

Reply:
Good call, I just tested and it does not clean out those dirs. I'll add some code to clean those out.